### ISSUE TITLE
Consider to set an aria label for each quick pick when it becomes active

### ISSUE THREAD

**Issue by weinand** [microsoft/vscode#93947]
Refs #93648 
Orca 3.28.0 on Ubuntu 18.04.4 LTS

- open a quickpick (e.g. file quickpick or command quickpick)

Screen reader says something like "type to narrow down ??? combo box" (I couldn't understand the word "???").
Instead of saying "combo box" quickpick could be more specific and say "files" or "commands".

[I have no experience with screen readers. I just report what I noticed.
Someone who has to rely on a screen reader should test this and provide feedback.]

**Comment by chrmarti** [comment]
We could add an aria label that each use of the QuickPick would need to supply. Alternatively we typically have a placeholder that is context specific (but might not always speak for itself from what I see).

**Comment by bpasero** [comment]
I am using `placeholder` exactly for that reason and was under the assumption that screen readers would read it out to the user. 

@chrmarti for the aria label I think this needs infrastructure on the level of quick pick, I only see the `ariaLabel` attribute for items.

Moving to April, would also like to hear @isidorn input on this and potentially real screen reader users.

**Comment by isidorn** [comment]
The placeholder should be read. Absolutely agree on that.
I now tried it with VoiceOver and the placeholder does not get read. 
@bpasero I see that you set the `ariaLabel` to the "Type to narrow down results". Why not simple set it to the placeholder message?

@marcozehe @jvesouza @ajborka feedback very welcome on what should be read when the quick pick gets opened (for example via cltr/cmd + p)

![Screenshot 2020-04-01 at 13 55 36](https://user-images.githubusercontent.com/1926584/78134454-7a6d9980-7420-11ea-838a-ebc2de098536.png)

**Comment by weinand** [comment]
With "F1" screen reader now says:
"Leaving main content. Type the name of a command to run editable combo box. Opens menu".

For "Cmd-P" screen reader reads the placeholder text (which is a bit long but gets the message across).

**Comment by jvesouza** [comment]
LOL! It was 'Show de bola' as we say here in Brazil, when something is very good.

When I press ctrl + p, orca announces the following:
Search files by name (append : to go to line or @ to go to symbol)

I think I'm falling in love with VSCode. I hope my wife doesn't read this since she doesn't know what VSCode is.
